File: bigbangarif.py
import json

bigbang = []

for i in range (1,101):
    if i % 3 == 0 and i % 5 == 0:
        bigbang.append("BIGBANG")
    elif i % 3 == 0:
        bigbang.append("BIG")
    elif i % 5 == 0:
        bigbang.append("BANG")
    else:
        bigbang.append(str(i))

print(bigbang)

with open("output.json", "w") as f:
    json.dump(bigbang, f)
    

File: bigbangarif.js
const fs = require("fs");
const bigbang = [];

for (let i=1; i<=100; i++) {
    if (i%3 === 0 && i%5 == 0) {
        bigbang.push("BIGBANG");
    } else if (i%3 === 0) {
        bigbang.push("BIG");
    } else if (i%5 === 0) {
        bigbang.push("BANG");
    } else {
        bigbang.push(i.toString());
    }
}

console.log(bigbang);
fs.writeFileSync("output.json", JSON.stringify(bigbang, null, 2));
console.log("\noutput.json has been created!!!")